hedera-node/src/main/java/com/hedera/services/tokens/HederaTokenStore.java

```diff
@@ -249,27 +229,34 @@ public ResponseCodeEnum adjustBalance(AccountID aId, TokenID tId, long adjustmen
 	}
 
 	@Override
-	public ResponseCodeEnum wipe(AccountID aId, TokenID tId, boolean skipKeyCheck) {
+	public ResponseCodeEnum wipe(AccountID aId, TokenID tId, long wipingAmount, boolean skipKeyCheck) {
 		return sanityChecked(aId, tId, token -> {
```

Comment:
IMO there should be a check that `wipingAmount` cannot be zero. There is no point in doing wipe with `0`